docs/data-management.rst

```diff
@@ -130,6 +132,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
```

Comment:
either go for ``Datasets/Projects`` or ``Dataset/Project``

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/592c43f536e4835707f593d062df3b1d016f1ff7